styles/Editor.css

```diff
@@ -958,8 +961,28 @@
 .ProseMirror .table-footnotes-element {
   counter-reset: table_footnote;
 }
+.delete-footnote-dialog button:first-child {
+  width: 96px;
+}
+.delete-footnote-dialog button:not(:first-child) {
+  width: 79px;
```

Comment:
Why the button have different widths?

Reply:
It was noted in the QA test that the design has a wider width, which was flagged as an issue.

Reply:
https://www.figma.com/file/vspHhzKTbwSkZWQWOcmIw2/LW---Editor-%26-Roles?node-id=25316%3A373326&mode=dev

Reply:
They are kidding :joy:

Please remove all these specifications. We are using a specific themed dialog across our editor. If they want to change how it looks, this change should live in the style guide, not here. We will not do that to this specific dialog.

Reply:
Would you kindly share this on [LEAN-3143](https://jira.atypon.com/browse/LEAN-3143) and seek approval from Panos to prevent the QA team from reporting this as an issue again, prior to undoing it?
and what other specifications that need to be removed rather than this change

Reply:
I just meant this and the previous `.delete-footnote-dialog button:first-child` selectors. Please disregard, as this change shouldn't be made here anyway. I will add a comment on the ticket.

Reply:
done